airbyte-ci/connectors/pipelines/tests/test_tests/test_common.py

```diff
@@ -243,116 +243,3 @@ async def test_params(self, dagger_client, mocker, test_context_ci, test_input_d
         assert set(acceptance_test_step.params_as_cli_options) == {"-ra", "--disable-warnings", "--durations=3"}
         acceptance_test_step.extra_params = {"--durations": ["5"], "--collect-only": []}
         assert set(acceptance_test_step.params_as_cli_options) == {"-ra", "--disable-warnings", "--durations=5", "--collect-only"}
-
-
-class TestCheckBaseImageIsUsed:
```

Comment:
I don't see any usage of `CheckBaseImageIsUsed` and we are removing the tests. Should we remove [the implementation](https://github.com/airbytehq/airbyte/blob/414b9aded25fd14e0253c97242a3c2f68f801f79/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py#L288-L312) as well?

Reply:
Oh yes good catch, I'll remove the implementation.